testbed/tests/widgets/test_slider.py

```diff
@@ -78,13 +80,14 @@ async def test_value(widget, probe, on_change):
 def assert_set_value(widget, value_in, value_out=None):
     if value_out is None:
         value_out = value_in
-    value_out = assert_set_get(widget, "value", value_in, value_out)
+    value_out = assert_set_get(widget, "value", value_in, pytest.approx(value_out))
```

Comment:
A mistake on my part. In porting from range to min/max, I missed that the iOS implementation was maintaining a shadow copy of the range because of 32 bit precision.